scripts/launch-xrd

```diff
@@ -244,10 +245,6 @@ while [[ $# -gt 0 ]]; do
             ;;
         -- )
             shift
-            if [[ ${#CTR_MGR_ARGS[@]} -ne 0 ]]; then
-                echo "When using --, use container manager args after it. These args are inavlid for the script ${CTR_MGR_ARGS[*]}" >&2
-                bad_usage
-            fi
```

Comment:
Now I think we should give an error if `$#` is 0, since I can't see why we'd want to allow `launch-xrd IMG --` (`--` should always come before `IMG` so that it doesn't get confused with passing a command for the container to run, similar to `kubectl exec CTR -- CMD`.